src/sentry/static/sentry/app/components/contextPickerModal.tsx

```diff
@@ -228,15 +222,24 @@ class ContextPickerModal extends React.Component<Props> {
       .filter(({status}) => status.id !== 'pending_deletion')
       .map(({slug}) => ({label: slug, value: slug}));
 
+    const customStyles = {
+      option: provided => ({
+        ...provided,
+        borderTop: `1px solid ${theme.gray1}`,
+      }),
+      menuList: provided => ({
+        ...provided,
+        paddingTop: '0px',
```

Comment:
```suggestion
        paddingTop: 0,
```

I believe we don't need to specify units when the value is 0